data/sql/updates/pending_db_world/rev_1645624680895656000.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1645624680895656000');
+
+-- Add gossip flag to Expedition Commander, previously was 0
+UPDATE creature_template SET npcflag = 1 WHERE entry IN (33210,34254);
+-- Assign the proper? menuID
+UPDATE `creature_template` SET `gossip_menu_id` = 10314 WHERE `entry` IN (33210,34254);
```

Comment:
```suggestion
UPDATE `creature_template` SET `gossip_menu_id` = 10314 WHERE `entry` IN (33210,34254);

```
It is best style to have a newline character at end of the file.

Reply:
Although I think the Github suggestion is actually added two newlines, so you may need to fix that. Haha.

Reply:
Thanks! 2x